tests/system/marathon_common_tests.py

```diff
@@ -1154,3 +1157,173 @@ def test_ipv6_healthcheck(docker_ipv6_network_fixture):
         "The number of healthy tasks is {}, but {} was expected".format(app['tasksHealthy'], target_instances_count)
 
     client.remove_app(app['id'], True)
+
+@shakedown.dcos_1_11
+@pytest.mark.skipif("shakedown.ee_version() is None")
```

Comment:
do faultdomains really work in EE only?

Reply:
I think I read it somewhere. Let me verify.

Reply:
Actually I can't find any document that explicitly states that this is an EE feature. 
So I can remove this condition.

Reply:
I think I had in mind the related _Cluster Linking_ that is an EE feature.